core/unix/os.c

```diff
@@ -8637,13 +8688,19 @@ post_system_call(dcontext_t *dcontext)
     }
 #endif
 #ifdef LINUX
+#    ifdef SYS_ppoll_time64
+    case SYS_ppoll_time64:
```

Comment:
Add a test to suite/tests/linux/syscall_pwait.cpp (maybe just the raw version, a la `psyscall_raw_ppoll`) under ifdef SYS_ppoll_time64?

Reply:
Done.